apps/payments/next/app/[locale]/[offeringId]/[interval]/checkout/[cartId]/error/page.tsx

```diff
@@ -21,16 +21,32 @@ import {
 import { config } from 'apps/payments/next/config';
 import type { Metadata } from 'next';
 import { CartErrorReasonId } from '@fxa/shared/db/mysql/account';
+import { buildPageMetadata } from '@fxa/payments/ui';
 
 // forces dynamic rendering
 // https://nextjs.org/docs/app/api-reference/file-conventions/route-segment-config
 export const dynamic = 'force-dynamic';
 
-export const metadata: Metadata = {
-  title: 'Error',
-  description:
-    'There was an error processing your subscription. If this problem persists, please contact support.',
-};
+export async function generateMetadata(
+  {
+    params,
+    searchParams,
+  }: {
+    params: CheckoutParams;
+    searchParams: Record<string, string> | undefined;
+  },
+): Promise<Metadata> {
+  return buildPageMetadata(
+      params,
+      'Error',
+      'There was an error processing your subscription. If this problem persists, please contact support.',
+      'error',
+      'checkout',
```

Comment:
Do these need to be localized, or are they localized elsewhere?

Reply:
Oh yes, good catch. This totally skipped my mind, even though it is the first requirement in the ticket. Sorry about that.

Do we have a follow up ticket for this @elizabeth-ilina